google-cloud-spanner/src/test/java/com/google/cloud/spanner/connection/AbstractConnectionImplTest.java

```diff
@@ -993,6 +993,18 @@ private void testExecuteUpdate(final StatementType type) {
             },
             connection,
             ErrorCode.FAILED_PRECONDITION);
+      } else if (type == StatementType.DDL) {
```

Comment:
This change also requires that we re-generate the generated tests. You can do that with the following Maven command:

```
mvn -Ddo_log_statements=true exec:java -Dexec.mainClass=com.google.cloud.spanner.connection.SqlTestScriptsGenerator -Dexec.classpathScope="test"
```

See https://github.com/googleapis/java-spanner/blob/27ef53c8447bd51a56fdfe6b2b206afe234fad80/google-cloud-spanner/src/test/java/com/google/cloud/spanner/connection/ConnectionImplGeneratedSqlScriptTest.java#L39

Reply:
This is done. Thanks for pointing it out.